addonscript/requirements/base.py37.txt

```diff
@@ -195,84 +206,108 @@ multidict==5.1.0 \
+    --hash=sha256:e4fac90784481d221a8e4b1162afa7c47ed953be40d31ab4629ae917510051df \
+    --hash=sha256:fa5ae20527d8e831e8230cbffd9f8fe952815b2b7dae6ffec25318803a7528fc
     # via scriptworker
-requests==2.25.0 \
-    --hash=sha256:7f1a0b932f4a60a1a65caa4263921bb7d9ee911957e0ae4a23a6dd08185ad5f8 \
-    --hash=sha256:e786fa28d8c9154e6a4de5d46a1d921b8749f8b74e28bde23768e5e16eece998 \
-    # via github3.py, taskcluster
-rsa==4.6 \
-    --hash=sha256:109ea5a66744dd859bf16fe904b8d8b627adafb9408753161e766a92e7d681fa \
-    --hash=sha256:6166864e23d6b5195a5cfed6cd9fed0fe774e226d8f854fcb23b7bbef0350233 \
+requests==2.25.1 \
+    --hash=sha256:27973dd4a904a4f13b263a19c866c13b92a39ed1c964655f025f3f8d3d75b804 \
+    --hash=sha256:c210084e36a42ae6b9219e00e48287def368a26d03a048ddad7bfee44f75871e
+    # via
+    #   github3.py
+    #   taskcluster
+rsa==4.7 \
+    --hash=sha256:69805d6b69f56eb05b62daea3a7dbd7aa44324ad1306445e05da8060232d00f4 \
+    --hash=sha256:a8774e55b59fd9fc893b0d05e9bfc6f47081f46ff5b46f39ccf24631b7be356b
     # via python-jose
-scriptworker==36.0.1 \
-    --hash=sha256:43ceb536d262a76c476456e9c6e9715014da2041b76df6a7c9af2fd108100ede \
-    --hash=sha256:eb68ff829408c71e95c9b8bcdaee7c7b14b249fd5d7d484bd8b7fe3a2bcedb08 \
+scriptworker==36.0.5 \
+    --hash=sha256:28787a69993f65bb1d7e71e1d22d428ad72e17873b0a93a4f22b3b33ad807dd1 \
+    --hash=sha256:6eb373931b628e320a9cb14b949bd0070ef3f7f85ef7f396022f07e70c12988f
     # via -r requirements/base.in
 six==1.15.0 \
     --hash=sha256:30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259 \
-    --hash=sha256:8b74bedcbbbaca38ff6d7491d76f2b06b3592611af620f8426e82dddb04a5ced \
-    # via cryptography, ecdsa, jsonschema, mohawk, python-dateutil, python-jose, taskcluster
+    --hash=sha256:8b74bedcbbbaca38ff6d7491d76f2b06b3592611af620f8426e82dddb04a5ced
+    # via
+    #   ecdsa
+    #   jsonschema
+    #   mohawk
+    #   python-dateutil
+    #   python-jose
+    #   taskcluster
 slugid==2.0.0 \
     --hash=sha256:a950d98b72691178bdd4d6c52743c4a2aa039207cf7a97d71060a111ff9ba297 \
-    --hash=sha256:aec8b0e01c4ad32e38e12d609eab3ec912fd129aaf6b2ded0199b56a5f8fd67c \
+    --hash=sha256:aec8b0e01c4ad32e38e12d609eab3ec912fd129aaf6b2ded0199b56a5f8fd67c
     # via taskcluster
 taskcluster-urls==13.0.1 \
     --hash=sha256:5e25e7e6818e8877178b175ff43d2e6548afad72694aa125f404a7329ece0973 \
     --hash=sha256:b25e122ecec249c4299ac7b20b08db76e3e2025bdaeb699a9d444556de5fd367 \
-    --hash=sha256:f66dcbd6572a6216ab65949f0fa0b91f2df647918028436c384e6af5cd12ae2b \
+    --hash=sha256:f66dcbd6572a6216ab65949f0fa0b91f2df647918028436c384e6af5cd12ae2b
     # via taskcluster
-taskcluster==39.1.1 \
-    --hash=sha256:608631462b881118a279e1825aa5b10745d8acefb612e72c6fdb1af728bcfae3 \
-    --hash=sha256:66820075f6f4b6911e4df0679c460451e86e203b73a07f9ccd0780f671822628 \
-    --hash=sha256:fbf05197991134fc541a4cc5676bb9eebbf409e61fa889aa38e40e38a73ab476 \
+taskcluster==41.0.0 \
```

Comment:
Hm. Our cluster is on 39.x.x and we're bumping the taskcluster client to 41.0.0. Asking to see if this will cause issues.

Reply:
It will, so I pinned tc to <40 in all relevant base.in files. We'll have to bump this pin manually when our cluster upgrades to 40. https://github.com/mozilla-releng/scriptworker/issues/485 will block us upgrading to 41.